content/en/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -324,7 +323,7 @@ reasons:
   forcing a restart, and the init container completion record has been lost due
   to {{< glossary_tooltip text="garbage collection" term_id="garbage-collection" >}}.
 
-The Pod will not be restarted when the init container image is changed, or the
+The Pod will _not_ be restarted when the init container image (source image at remote repository) is changed, or the
```

Comment:
The text in the parens are unclear and confusing.

Reply:
@tengqm, Can you guide what could be changes here should be?

Reply:
Just clarify what do you mean by "source image at remote repository".

Reply:
Hi @tengqm, Here ```source image at remote repository``` refers to images like redis, nginx etc. on the docker-hub or kubernetes images on the gcr.io.

Reply:
Why you want to highlight those images from the public container image registries?
Is there any difference for private registries?

Reply:
No, @tengqm I just have suggested these names as examples, images can be from private or public repo's, that's why I suggested `source images` and `remote repo's` as the name, if you suggest other names I will change it.

Reply:
I am afraid there is a misunderstanding here. What the original text meant was that the image changed from 'foo:1.0' to 'bar:1.0', but you are adding an extra, unnecessary parenthesis to make this confusing. If we are adding the texts as suggested by this PR, what we mean then becomes that the containers cannot be aware of image content changes at the registry side, especially when the images are from a remote, public registry.

Reply:
How about making “image” into a glossary tooltip? Or explaining that actually the thing being changed is the _image name_?

Reply:
I Have updated the PR, by adding image glossary as.